raml-parser-2/src/main/java/org/raml/v2/internal/impl/v10/phase/ExampleValidationPhase.java

```diff
@@ -68,6 +68,9 @@
 public class ExampleValidationPhase implements Phase
 {
     private ResourceLoader resourceLoader;
+    private static final String LIMIT_ERROR_MESSAGE_LENGTH = "raml.limit_error_message_length";
+    public static boolean limitErrorMessageLength = Boolean.valueOf(System.getProperty(LIMIT_ERROR_MESSAGE_LENGTH, "false"));
+    private static final int errorMessageMaxLength = 10000;
```

Comment:
I think that limiting the error message should be the default. Actually, I would add a numeric property more than a boolean one, so that, by default, the parser would respond up to either a predetermined number of errors or the amount set by the user. 
An option to get all error messages could be accepting a value like `All`
What're your thoughts?

Reply:
Done